benchmarking_sbi/run.py

```diff
@@ -24,7 +25,7 @@
 @hydra.main(config_path="config", config_name="config")
 def main(cfg: DictConfig) -> None:
     log = logging.getLogger(__name__)
-    log.info(cfg.pretty())
+    log.info(pformat(cfg))
```

Comment:
https://github.com/omry/omegaconf/issues/358 -- should we use `OmegaConf.to_yaml(cfg)` instead?

Reply:
make sense see https://github.com/sbi-benchmark/results/pull/3/commits/29ed3662c62f63208dcef3025c1d7c6d52fcc12f